cmds/exp/watch/watch.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// watch periodically executes the executable.
+//
+// Synopsis:
+//     watch [-n] SEC [-t] cmd-exec
+//
+// Description:
+//    cmd-exec is executed every n seconds
+//    example, watch -n 5 dmesg >> log.txt 
+//    : executes dmesg every 5 sec and stores the log in log.txt
+//
+// Options:
+//     -n: time in seconds
+//     -t: do not print header
+
```

Comment:
remove empty line, so that this help text is actually rendered as godoc. (must be immediately preceding package statement)